Automattic/pocket-casts-ios
# Playback 2023: show top categories

| 📘 Part of: #1142 | 🎨 Designs: [Figma](https://www.figma.com/file/lH66LwxxgG8btQ8NrM0ldx/End-of-Year?type=design&node-id=1599-20385&mode=design) |
|:---:|:---:|

Adds the 2023 visual for the Top Categories Story.

**Animations are not part of this task and will be tackled later.**

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/c3960704-57ca-4d8e-9876-67ba41ad9623" width="300">

## To test

1. Make sure you're logged in to an account that has a few episodes listened
2. Go to Profile > Settings > Beta Features > enable `endOfYear`
3. Go back to Profile
4. Tap the EOY image
5. Skip to the fifth story
5. ✅ Check that the story looks good, that it shows your top categories and for the top one, the number of episodes listened to
6. Tap "Share this story" > Save Image
7. Go to Photos
8. ✅ The story image asset should be the last one on Photos and should have your top categories
9. Check the stories again, but this time go to the fourth one (the top five podcasts)
10. Dismiss it
11. ✅ You shouldn't see the gradient when dismissing
12. Test the same with the fifth story — top categories

Please test on a big device (iPhone 15 Pro Max, for example), a normal device (iPhone 15), and a smaller device (iPod touch).

## Checklist
